src/cli.rs

```diff
@@ -26,18 +26,25 @@ pub struct Args {
     #[clap(long, default_value = "/run/postgresql")]
     pub db_host: DbHost,
 
-    /// Host for the cardano-db-sync database connection
-    #[clap(long, default_value = "/run/postgresql")]
-    pub db_pass: DbPass,
+    /// Password for the cardano-db-sync database connection
+    #[clap(long)]
+    pub db_pass: Option<DbPass>,
```

Comment:
does it break downstream without the password?

Reply:
I'm not 100% sure what you mean. The password is optional because postgres doesn't always require a password (e.g. my local instance accepts any connections without a password from localhost). It's mainly to make the format string easier